components/infobox/wikis/osu/infobox_league_custom.lua

```diff
@@ -0,0 +1,124 @@
+---
+-- @Liquipedia
+-- wiki=osu
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Game = require('Module:Game')
+local Lua = require('Module:Lua')
+local Logic = require('Module:Logic')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local League = Lua.import('Module:Infobox/League', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+local _MODES = {
+	standard = 'Standard[[Category:osu!standard Tournaments]]',
```

Comment:
```suggestion
	standard = {display = 'Standard', category = 'Osu!standard Tournaments'},
```
as suggestion for better format

Reply:
got **Module:Infobox/Widget/Cell/dev:68: bad argument #1 to 'wikitext' (string or number expected, got table)**

The function for modes currently is this one
![image](https://github.com/Liquipedia/Lua-Modules/assets/88981446/a22df474-6ab5-4f20-81f2-53fd6b22fa83)

Reply:
of course you need to adjust the usage
in the display part use `CustomLeague._getGameMode().display`

in the category function use `CustomLeague._getGameMode().category`